_problems/fee-bumping.md

```diff
@@ -23,30 +25,33 @@ Let's look at the implications of the current methods have on layer-2 protocols.
 ### Pre-signed Transactions
 
 Often the transaction an honest party is racing against the clock to confirm is a *pre-signed* transaction.
-It was signed when their counterparty was cooperative -- now the honest party needs to go on chain precisely because they are not cooperative.
+It was signed when their counterparty was cooperative -- now they need to go on chain precisely because they are not cooperative.
 If fees become persistently higher than the pre-signed transaction's fees the party may miss their deadline.
 The obvious but suboptimal solution is to monitor fees closely and update any time-sensitive pre-signed transactions with the counterparty while they are online (this method is available via the [`update_fee`] message in lightning today).
 
 It is much more desirable to allow the party to spontaneously increase the fee at will.
 This can be done with [CPFP].
-For example, in the [Lightning] protocol commitment transactions are designed with [anchor outputs] so they can be easily fee bumped through CPFP.
+For example, in the [Lightning] protocol commitment transactions have [anchor outputs] so they can be easily fee bumped through CPFP.
+Although this works it is rather complex to design and implement.
+Furthermore, users have to keep unattached fee-bumping UTXOs around in their wallet in case their need to do a fee-bump via CPFP.
 
 ### Resource Usage of Replacement
 
-Allowing a mempool transaction to be replaced by higher fee paying transactionrmay open up Bitcoin network nodes to *denial of service* (DoS) attacks.
+Allowing a mempool transaction to be replaced by higher fee paying transaction may open up Bitcoin network nodes to *denial of service* (DoS) attacks.
 Without replacement, a malicious party's ability to get your node to process a transaction is limited by the number of UTXOs they own.
-With replacement an attacker can repeatedly spam your node with transactions that replace previous ones even if they only own a single UTXO.
+With replacement an attacker can repeatedly spam your node with transactions that replace previous ones even if they only own a single UTXO forcing you to spend time verifying them.
```

Comment:
This can be a bit clearer, "forcing you to adopt a reactive behavior , stay online and rebroadcast your transactions with an increasing frequency as the contentious timelock expiration approaches"

Reply:
This point about CPU/bandwidth consumption. It is trying to explain the concerns with adding replacement in general for the p2p network i.e. All Bitcoin nodes are susceptible to DoS attacks if Tx replacement rules are not carefully designed.

I believe your point is about your counterparty replacing your transactions. I actually don't understand in which situation this is a concern? I mean if they replace your transaction then by the protocol rules it must be valid and paying a higher fee. e.g. if you are broadcasting a commit tx and they replace it with their own that pays a higher fee that is not really a security concern.